src/modules/navigator/navigator_vtol_params.c

```diff
@@ -0,0 +1,75 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2018 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * AS IS AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/**
+ * @file navigator_vtol_params.c
+ *
+ * VTOL parameters currently required by Navigator.
+ * FIXME:
+ * - These are only here because the VTOL modules are not built on all boards
+ *   that use the Navigator
+ * - Navigator should not have to use these
+ *
+ */
+
+/**
+ * Approximate deceleration during back transition
+ *
+ * The approximate deceleration during a back transition in m/s/s
+ * Used to calculate back transition distance in mission mode. A lower value will make the VTOL transition further from the destination waypoint.
+ *
+ * @unit m/s/s
+ * @min 0.00
+ * @max 20.00
+ * @increment 1
+ * @decimal 2
+ * @board px4fmu-v2
+ * @group VTOL Attitude Control
+ */
+PARAM_DEFINE_FLOAT(VT_B_DEC_MSS, 2.0f);
+
+
+/**
+ * Delay in seconds before applying back transition throttle
+ * Set this to a value greater than 0 to give the motor time to spin down.
+ *
+ * unit s
+ * @min 0
+ * @max 10
+ * @increment 1
+ * @decimal 2
+ * @board px4fmu-v2
```

Comment:
board? This definitely isn't specific to fmu-v2.

Reply:
Look at the commit: https://github.com/PX4/Firmware/pull/8789/commits/0a4f756d1dad5579395347cb748e62262bc8c03d. The goal for now is to fix missing params and we can find a nice solution later.

Reply:
Yes I saw, but still didn't quite understand. These VTOL params are only used in navigator, so I'd say they need to live in navigator completely.

If we had a bit more control we could only mark them active if a VTOL, but really I'd say this is a clear sign of bad architecture. The better solution would be handling this entirely in VTOL and publishing something, similar to the way FW acceptance radius works now. @RomanBapst 

For now does moving them entirely to navigator solve the problem?

Reply:
Turns out `modules/vtol_att_control` was already in v2, and the failing build was due to flash overflow, and not missing params. I removed the file accordingly.